Lib/typing.py

```diff
@@ -676,7 +676,7 @@ def __init__(self, arg, is_argument=True, module=None, *, is_class=False):
         self.__forward_evaluated__ = False
         self.__forward_value__ = None
         self.__forward_is_argument__ = is_argument
-        self.__forward_is_class__ = is_class
+        self.__forward_allow_special_forms__ = is_class
```

Comment:
I think we should leave this one alone too.

Reply:
changed